README.md

```diff
@@ -80,6 +80,10 @@ We've provided some very, very basic recipes and are interested in adding more,
 so please send a pull request if you think anything would serve the larger
 community.
 
+## More information
```

Comment:
We should have a subproject summary, but the deployment scripts should really be front & center in the readme, so that a new user can get started ASAP.